### PR TITLE
Content layout - List of tech-box for front and back sections with flex-wrap

### PR DESCRIPTION
**Pull request by rsuescun**
## Summary
- Added tech-box-list style to show box-tech in flex wrap
- Added all tech logos

## Screenshot
image

## Trello card
https://trello.com/c/u3Gjxgzy/23-content-layout

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,26 +9,101 @@
...
-      <img class="tech-box-logo" src="assets/tech/react_icon.svg" alt="Logo React">
-      <h2 class="tech-box-title">ReactJS</h2>
-      <span class="tech-box-launch">Release: 2013</span>
-      <p class="tech-box-text">
-        React is a declarative, component-based JavaScript library for building user interfaces.
-      </p>
+  <div class="tech-box-list">
+    <div class="tech-box">
+      <div class="tech-box-content">
+        <img class="tech-box-logo" src="assets/tech/react_icon.svg" alt="Logo React">
```
Comment (FrankIglesias) [inline index.html]: add slash before >

File: index.html [hunk 2]
```diff
@@ -9,26 +9,101 @@
...
+        <h2 class="tech-box-title">ReactJS</h2>
+        <span class="tech-box-launch">Release: 2013</span>
+        <p class="tech-box-text">
+          React is a declarative, component-based JavaScript library for building user interfaces.
+        </p>
+      </div>
+    </div>
+    <div class="tech-box">
+      <div class="tech-box-content">
+        <img class="tech-box-logo" src="assets/tech/angular_icon.svg" alt="Logo AngularJS">
```
Comment (FrankIglesias) [inline index.html]: add slash before >

File: index.html [hunk 3]
```diff
@@ -9,26 +9,101 @@
...
+        <h2 class="tech-box-title">AngularJS</h2>
+        <span class="tech-box-launch">Release: 2010</span>
+        <p class="tech-box-text">
+          AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architecture.
+        </p>
+      </div>
+    </div>
+    <div class="tech-box">
+      <div class="tech-box-content">
+        <img class="tech-box-logo" src="assets/tech/vue_icon.svg" alt="Logo VueJS">
```
Comment (FrankIglesias) [inline index.html]: add slash before >

File: index.html [hunk 4]
```diff
@@ -9,26 +9,101 @@
...
+        <h2 class="tech-box-title">VueJS</h2>
+        <span class="tech-box-launch">Release: 2014</span>
+        <p class="tech-box-text">
+          Vue.js (pronounced /vjuː/, like view) is an open-source JavaScript framework for building user interfaces.
+        </p>
+      </div>
+    </div>
+    <div class="tech-box">
+      <div class="tech-box-content">
+        <img class="tech-box-logo" src="assets/tech/elm_icon.svg" alt="Logo Elm">
```
Comment (FrankIglesias) [inline index.html]: add slash before >

File: index.html [hunk 5]
```diff
@@ -9,26 +9,101 @@
...
+        <h2 class="tech-box-title">Elm</h2>
+        <span class="tech-box-launch">Release: 2016</span>
+        <p class="tech-box-text">
+          Elm is a domain-specific programming language for declaratively creating web browser-based graphical user interfaces.
+        </p>
+      </div>
+    </div>
+    <div class="tech-box">
+      <div class="tech-box-content">
+        <img class="tech-box-logo" src="assets/tech/polymer_icon.svg" alt="Logo Polymer">
```
Comment (FrankIglesias) [inline index.html]: add slash before >

File: index.html [hunk 6]
```diff
@@ -9,26 +9,101 @@
...
-      <h2 class="tech-box-title">ReactJS</h2>
-      <span class="tech-box-launch">Release: 2013</span>
-      <p class="tech-box-text">
-        React is a declarative, component-based JavaScript library for building user interfaces.
-      </p>
+  <h1 class="title">Back-End Tools</h1>
+  <div class="tech-box-list">
+    <div class="tech-box">
+      <div class="tech-box-content">
+        <img class="tech-box-logo" src="assets/tech/rails_icon.png" alt="Logo Ruby on Rails">
```
Comment (FrankIglesias) [inline index.html]: add slash before >

File: index.html [hunk 7]
```diff
@@ -9,26 +9,101 @@
...
+        <h2 class="tech-box-title">Ruby on Rails</h2>
+        <span class="tech-box-launch">Release: 2015</span>
+        <p class="tech-box-text">
+          Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.
+        </p>
+      </div>
+    </div>
+    <div class="tech-box">
+      <div class="tech-box-content">
+        <img class="tech-box-logo" src="assets/tech/express_icon.svg" alt="Logo NodeJS">
```
Comment (FrankIglesias) [inline index.html]: add slash before >

File: index.html [hunk 8]
```diff
@@ -9,26 +9,101 @@
...
+        <h2 class="tech-box-title">NodeJS</h2>
+        <span class="tech-box-launch">Release: 2009</span>
+        <p class="tech-box-text">
+          Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event-driven, non-blocking I/O model taht makes it lightweight and efficient.
+        </p>
+      </div>
+    </div>
+    <div class="tech-box">
+      <div class="tech-box-content">
+        <img class="tech-box-logo" src="assets/tech/aspnet_icon.png" alt="Logo ASP.NET Core">
```
Comment (FrankIglesias) [inline index.html]: add slash before >

File: index.html [hunk 9]
```diff
@@ -9,26 +9,101 @@
...
+        <h2 class="tech-box-title">ASP.NET Core</h2>
+        <span class="tech-box-launch">Release: 2016</span>
+        <p class="tech-box-text">
+          ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and the cross-platform .NET Core.
+        </p>
+      </div>
+    </div>
+    <div class="tech-box">
+      <div class="tech-box-content">
+        <img class="tech-box-logo" src="assets/tech/django_icon.svg" alt="Logo Django">
```
Comment (FrankIglesias) [inline index.html]: add slash before >

File: scss/tech.scss [hunk 10]
```diff
@@ -1,68 +1,73 @@
...
-    top: $tech-box-content-displace;
-    width: $tech-box-width;
-  }
+    &::before {
+      content: '';
+      background-color: $white;
+      border: 1px solid $green;
+      border-radius: $tech-box-border-radius;    
+      height: $tech-box-height;
+      left:0;
```
Comment (FrankIglesias) [inline scss/tech.scss]: add space before the 0

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

File: scss/tech.scss [hunk 11]
```diff
@@ -1,55 +1,61 @@
...
   display: flex;
-  flex-direction: column;
-  height: $tech-box-height;
-  margin: 35px;
-  padding: 40px;
-  position: relative;
-  text-align: center;
-  width: $tech-box-width;
+  flex-wrap: wrap;
+  padding-left: 50px;
```
Comment (jmtov) [inline scss/tech.scss]: Try to make paddings and margins as symmetric as possible. This can help you handle LTR and RTL writing systems in the best way! 🙂

**Review by jmtov — APPROVED** [review]
Excellent job! 🤙I just left a small comment. 😺

File: index.html [hunk 12]
```diff
@@ -9,15 +9,84 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body>
+<body class="padding-app">
```
Comment (mmamaniwolox) [inline index.html]: This is not very descriptive, you can write something like `app-container` or someting like that

File: index.html [hunk 13]
```diff
@@ -9,15 +9,84 @@
...
+    <div class="tech-box">
+      <img class="tech-box-logo" src="assets/tech/react_icon.svg" alt="Logo React"/>
+      <h2 class="tech-box-title">ReactJS</h2>
+      <span class="tech-box-launch">Release: 2013</span>
+      <p class="tech-box-text">
+        React is a declarative, component-based JavaScript library for building user interfaces.
+      </p>
+    </div>
+    <div class="tech-box">
+      <img class="tech-box-logo" src="assets/tech/angular_icon.svg" alt="Logo AngularJS"/>
```
Comment (mmamaniwolox) [inline index.html]: Same than previous PR, in english the word `logo` should be after the name of the tech

File: index.html [hunk 14]
```diff
@@ -9,15 +9,84 @@
...
+    <div class="tech-box">
+      <img class="tech-box-logo" src="assets/tech/polymer_icon.svg" alt="Logo Polymer"/>
+      <h2 class="tech-box-title">Polymer</h2>
+      <span class="tech-box-launch">Release: 2015</span>
+      <p class="tech-box-text">
+        Polymer is an open-source JavaScript library for building web applications using Web Components.
+      </p>
+    </div>
+  </div>
+  <h1 class="title">Back-End Tools</h1>
```
Comment (mmamaniwolox) [inline index.html]: It is supposed to be only one `h1` in a single screen, you should use `h2`

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**